spring-native/storage-blob-native/README.md

```diff
@@ -175,10 +175,10 @@ GraalVM `native-image` compiler should be installed, see [System Requirements](h
 
 - Build the native application
 
-Run `mvn -Pnative -DskipTests package` command using `x64 Native Tools Command Prompt`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
+Run `mvn -P spring-cloud-azure-4.x,native-4.x -DskipTests package` command using `x64 Native Tools Command Prompt`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
 
 ```shell
-mvn -Pnative -DskipTests package
+mvn -P spring-cloud-azure-4.x,native-4.x -DskipTests package
```

Comment:
All these descriptions are about native-4.x, we lack content about native-6.x.

Reply:
We will not recommend users use these features on Spring Cloud Azure 6.0, the extra xxx-6.x profiles are used to simulate the profile `native` and `nativeTest` in spring-boot-starter-parent:3.0.0.

Reply:
could we add the recommends to this docs?

Reply:
This sample has been marked as not supporting Spring Cloud Azure 6.0, we will not say the native support with Spring Cloud Azure 6.0, but some samples actually can be built to executable with Spring Cloud Azure 6.0.